src/Tactic/DeriveComp.agda

```diff
@@ -4,28 +4,25 @@ module Tactic.DeriveComp where
 
 open import Prelude
 open import PreludeMeta hiding (TC) renaming (TCI to TC)
+open import MetaPrelude using (zipWithIndex)
```

Comment:
We now have `PreludeMeta` and `MetaPrelude`. There's probably some improvement to this structure that could be done at some point.